docs/index.asciidoc

```diff
@@ -84,6 +108,38 @@ output this value into a field called `clock`
 Otherwise, this value will be used verbatim as the event message. It
 will output this value into a field called `message`
 
+NOTE: Usage of `epoch`  and `sequence` in `message` setting is deprecated.
+Consider to use <<plugins-{type}s-{plugin}-sequence>> settings, which takes precedence
+over the usage of `message setting as selector.
+
+NOTE: If <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is enabled
+and `message` contains `epoch` or `sequence` it's ignored and it's not present as a field in
+the generated event.
+
+
+[id="plugins-{type}s-{plugin}-sequence"]
+===== `sequence`
+
+* Value can be any of: `none`, `epoch`, `sequence`
+* Default value is `"none""`
+
+If you set this to `none` then no sequence field is added.
+
+If you set this to `epoch` then this plugin will use the current
+timestamp in unix timestamp (which is by definition, UTC).  It will
+output this value into a field called `clock` if <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>>
+is disabled else the field name is `[event][sequence]`.
+
+If you set this to `sequence` then this plugin will send a sequence of
+numbers beginning at 0 and incrementing each interval.  It will
+output this value into a field called `clock` if <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>>
+is disabled else the field name is `[event][sequence]`
```

Comment:
```suggestion
output this value into a field called `clock` if <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>>
is disabled. Otherwise, the field name is `[event][sequence]`
```